### PR TITLE
doc section about findReact

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
//cc @kirovboris @VasilyStrelyaev

### CODE REVIEW

**Comment by eliw00d** [comment]
Can this be combined with find or non-React selector chains? For example:
```
let component = ReactSelector('Component')
    .find('div').withAttributes('class', /Class/)
    .findReact('SubComponent')
    .withProps({prop: 'prop'})
```
or can withProps be used with regex?
```
let component = ReactSelector('Component')
    .findReact('div').withProps('className', /Class/)
    .findReact('SubComponent')
    .withProps({prop: 'prop'})
```
The React DOM looks like:
```
<Component>
    <div className="Class-content-123">
        <SubComponent1>
        </SubComponent1>
        <SubComponent2>
        </SubComponent2>
    </div>
    <div className="Class-actions-456">
        <SubComponent3>
        </SubComponent3>
    </div>
</Component>
```

**Comment by kirovboris** [comment]
@eliw00d, Sure, it should work in all mentioned combinations. 
@MargaritaLoseva let's add this info to the docs.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,35 @@ const element = ReactSelector('componentName').withProps({
...
+
+You can call the `findReact` method in a chain, for example:
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+const itemsCount = ReactSelector('TodoApp').findReact('TodoList').findReact('span');
+```
+
+You can also combine `.findReact` with regular selectors and [other](](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#functional-style-selectors)) methods like `.find` or `.withText`, for example:
```
Comment (VasilyStrelyaev) [inline README.md]: > [other](](

must be something wrong with the link

File: README.md [hunk 2]
```diff
@@ -72,6 +72,35 @@ const element = ReactSelector('componentName').withProps({
...
+
+```js
+import ReactSelector from 'testcafe-react-selectors';
+
+var itemsCount = ReactSelector('TodoApp').find('.items-count').findReact('span');
+```
+
```
Comment (VasilyStrelyaev) [inline README.md]: From what is said in this topic, I don't understand the difference between `findReact` and `find` used for `ReactSelector`. Is it that `findReact` can search for components in addition to elements?

Comment (VasilyStrelyaev) [reply README.md]: What's going to happen if we write 
```
var itemsCount = ReactSelector('TodoApp').find('.items-count').find('span');
```

Comment (kirovboris) [reply README.md]: In this case you will look for `.items-count span` elements inside of `TodoApp` component

**Pull request merged**